feature-libs/customer-ticketing/occ/adapters/occ-customer-ticketing.adapter.spec.ts

```diff
@@ -20,6 +23,10 @@ const MockOccModuleConfig: OccConfig = {
         getTicket: 'users/${customerId}/tickets/${ticketId}',
         createTicketEvent: 'users/${customerId}/tickets/${ticketId}/events',
         getTickets: 'users/${customerId}/tickets',
+        uploadAttachment:
+          'users/${customerId}/tickets/${ticketId}/events/${eventCode}/attachments',
+        downloadAttachment:
+          'users/${customerId}/tickets/${ticketId}/events/${eventCode}/attachments/${attachmentId}',
```

Comment:
Is it possible to reuse the list of endpoints already defined here:

https://github.com/SAP/spartacus/blob/epic/customer-ticketing/feature-libs/customer-ticketing/occ/config/default-occ-customer-ticketing-config.ts

Reply:
nope, because the endpoints needs to be imported as `OccEndpoints`, I tried importing it directly. It does not work that way.